src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Performance/PreferCountOverAnyTests.cs

```diff
@@ -0,0 +1,286 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Testing;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Performance.PreferLengthCountIsEmptyOverAnyAnalyzer,
+    Microsoft.NetCore.CSharp.Analyzers.Performance.CSharpPreferLengthCountIsEmptyOverAnyFixer>;
+using VerifyVB = Test.Utilities.VisualBasicCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Performance.PreferLengthCountIsEmptyOverAnyAnalyzer,
+    Microsoft.NetCore.VisualBasic.Analyzers.Performance.BasicPreferLengthCountIsEmptyOverAnyFixer>;
+
+namespace Microsoft.NetCore.Analyzers.Performance.UnitTests
+{
+    public class PreferCountOverAnyTests
+    {
+        private const string CSharpTemplate = @"
+using System.Collections.Generic;
+using System.Linq;
+
+public class Tests {{
+    {0}
+}}";
+
+        private const string VbTemplate = @"
+Imports System.Collections.Generic
+Imports System.Linq
+
+Public Class Tests
+    {0}
+End Class";
```

Comment:
I personally prefer to avoid these.

Reply:
What would be a preferred approach? Should I put these in each test method?

Reply:
IMO, yes. I don't see the duplication as a problem.